runners/gradle-plugin/src/main/kotlin/org/jetbrains/dokka/gradle/GradleDokkaSourceSetBuilder.kt

```diff
@@ -1,12 +1,10 @@
 @file:Suppress("FunctionName", "UnstableApiUsage")
+@file:JvmName("GradleDokkaSourceSetBuilderKt")
```

Comment:
Just out of curiosity: why do we need this?

Reply:
I think this is not needed anymore. But I think we encountered some compiler incompatibility here. e.g. All extension functions in that file resulted in  "NoSuchMethodFoundError" (or how it is actually called)

Reply:
Removed it, as it was not necessary anymore! 
🦅 👁️

Reply:
✅